infrastructure/core/outputs.tf

```diff
@@ -39,3 +39,11 @@ output "core_kv_uri" {
 output "subnet_address_spaces" {
   value = local.subnet_address_spaces
 }
+
+output "deployer_ip_address" {
+  value = var.deployer_ip_address
+}
+
+output "local_mode" {
```

Comment:
You don't need to output this as it's defined as a global variable - instead you can add it to the top-level terragrunt.hcl file in the `inputs` section, then every module that inherits it will have access to it as a variable

Reply:
cheers! done